client.go

```diff
@@ -197,6 +201,14 @@ func FlushInterval(interval time.Duration) Option {
 	}
 }
 
+// SnapshotValidateInterval will return an option that will set the interval
+// at which RPC calls are made when validating that a snapshot has finished.
+func SnapshotValidateInterval(interval time.Duration) Option {
```

Comment:
do we really need this option? it seems like we should just use some sane default and not give extra burden on the client to even consider changing it.

Reply:
Removed the option. I'm not sure what a sane option is. I chose half a second here but I'm happy to change it.